### ISSUE TITLE
Update angular up to rc.5

### ISSUE THREAD

**Issue by alexpods** [alexpods/angular2-universal-starter#65]

**Comment by glebmachine** [comment]
When you plan to do the migration? 

I'm wondering to have hmr, but it requires rc.5 or higher

**Comment by ram-you** [comment]
And now the rc.6 is out.

**Comment by glebmachine** [comment]
yeah

**Comment by glebmachine** [comment]
Firstly, this issue should be resolved:
https://github.com/angular/universal/issues/511

Angular universal hasn't updated to support yet..

**Comment by alexpods** [comment]
@glebmachine @ram-you Yep. Going to do the migration when angular/universal is ready for this.

**Comment by glebmachine** [comment]
@alexpods Good news comes to those who waits
https://github.com/angular/universal/commit/05b8f57adfd79bcbf2035631e23d373aa2d2912f
^_^

**Comment by alexpods** [comment]
@glebmachine Yep, I know. Already started adding support to this starter.

**Comment by alexpods** [comment]
Though I'm a little bit busy these days. So expect full implementation on Saturday/Sunday (maybe earlier, but I can't promise)

**Comment by glebmachine** [comment]
Sounds great)

**Comment by khuezy** [comment]
and now 2.0.0 is out.

**Comment by alexpods** [comment]
Sorry, guys, but it's not ready yet:(
I worked on the weekend on updating all the stuff. Browser and WebWorker environments work fine. Didn't finish Node environment though.
Please, wait 1 more week.
